### ISSUE TITLE
[BUG] v0.1 still doesn't load on old iOS versions

### ISSUE THREAD

**Issue by andrewmooreio** [flixlix/power-flow-card-plus#150]
**Describe the bug**

Same issue faced with 0.0.9.6 https://github.com/flixlix/power-flow-card-plus/issues/77

The card no longer loads on my old iPad Air running iOS 12.5.7.

- v0.0.9.5 worked fine
- Standard v0.0.9.6 didn't work.
- Custom v0.0.9.6 js file provided in #77 worked
- v0.1 doesn't work.

This applies to all methods of accessing Home Assistant. The native app, Safari and Google Chrome. No issues on my iPhone running iOS 16.3.1 or any desktop platforms.

No other custom cards are impacted, I use mushroom cards extensively throughout, and they load without issue.

I've tried:

- Clearing browser cache (Safari) and frontend cache (app).
- Reinstalling app completely.
- Restarting iPad.
- Restarting Home Assistant server.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to dashboard containing power-flow-card-plus card.
2. See error "Custom element doesn't exist: power-flow-card-plus" where card should be.

**Expected behavior**
The card should load and display the power flow UI.

**Screenshots**
![IMG_0001](https://user-images.githubusercontent.com/20435317/232035370-5586d299-cf3c-40d1-92fe-84a087eaf14f.PNG)

**Smartphone (please complete the following information):**
 - Device: iPad Air (1st gen)
 - OS: iOS 12.5.7
 - Browser: Safari, Chrome, Home Assistant app

**Please make sure all of this applies and check the checkboxes, by writing "X" inside of them.**

- [X] I understand that this is a channel for reporting bugs, not a support forum (If you need help configuring the card, go to: [Discussions](https://github.com/flixlix/power-flow-card-plus/discussions)).

- [X] I have the latest version of the card installed.

- [X] I have cleared my browser cache **after** installing, updating and configuring the card.

If you require any more info, please let me know. The Home Assistant app logs don't show anything useful. Thanks.

**Comment by flixlix** [comment]
I'm sorry for the inconvenience, but unfortunately I don't know how to fix this issue. I can't test it since I don't own an iPad...

**Comment by andrewmooreio** [comment]
Okay, understood. I'll try to troubleshoot using the diff between the v0.0.9.6 version you gave me and the standard one. I'm a Python guy but I'll try to figure it out.

Leave this issue open and I'll report back.

Thanks.

**Comment by andrewmooreio** [comment]
I've done some testing, and I've discovered the issue is not with the code itself.

If I install v0.0.9.5 through HACS, then manually copy the [power-flow-card-plus.js](https://github.com/flixlix/power-flow-card-plus/releases/download/v0.1/power-flow-card-plus.js) v0.1 file. Everything works as expected. Whereas installing v0.1 directly through HACS results in the display issue.

My only thought is that it might be one of the dependencies that changed between v0.0.9.5 and v0.0.9.6, which isn't playing well with the older version of WebKit present in iOS <13. The following dependencies changed:

- home-assistant-js-websocket 7.0.3 > 8.0.1
- superstruct 1.0.3 (new dependency)

Does that sound plausible @flixlix? I may be on the wrong track. If you have any other ideas, please share.

**Comment by flixlix** [comment]
When installing the code manually, can you check your browser console and verify that the right card is being used? Having the HACS resource enabled might lead to conflicts between the manual and the HACS files

**Comment by andrewmooreio** [comment]
@flixlix I've confirmed the correct version of the card is being loaded.

The console shows v0.1 and all new functionality such as secondary info templates work.

**Comment by flixlix** [comment]
That is very weird, because the file in the release is the exact same file that is given to HACS...

**Comment by andrewmooreio** [comment]
Does HACS do anything else except install the js file? Such as installing JavaScript dependencies.

**Comment by flixlix** [comment]
No, HACS only uses the js file referenced in the release. The dependencies are all already inside this file

**Comment by andrewmooreio** [comment]
How odd. Well, thank you for the clarification.

I'll keep playing and see if I can get a better explanation.

**Comment by mharizanov** [comment]
I am subscribing as I observe the same on an iPad mini 2 living its fifth life as a HA control panel.

**Comment by flixlix** [comment]
@andrewmooreio Are you able to check your browser console?

**Comment by garnio** [comment]
Tried the change of ?? in the JS as described by burn in the 0.2.5 release but it fails to load on older iOS devices was there ever a fix implemented?
